src/observerClass.ts

```diff
@@ -153,7 +153,17 @@ function makeObservableProp(target: any, propName: string): void {
         configurable: true,
         enumerable: true,
         get: function() {
+            let prevReadState = false;
+
+            if (_allowStateReadsStart && _allowStateReadsEnd) {
```

Comment:
@mweststrate should we `import * as mobx` or this null check is enough?

Reply:
nope. this should work fine